flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/connectors/hive/HiveDynamicTableFactoryTest.java

```diff
@@ -66,6 +71,36 @@ public static void setup() {
         tableEnv.getConfig().setSqlDialect(SqlDialect.HIVE);
     }
 
+    @Test
+    public void testHiveTableSourceJobConfWithCredentials() throws Exception {
+        final Text hdfsDelegationTokenKind = new Text("HDFS_DELEGATION_TOKEN");
+        final Text hdfsDelegationTokenService = new Text("ha-hdfs:hadoop-namespace");
+        Credentials credentials = new Credentials();
+        credentials.addToken(
+                hdfsDelegationTokenService,
+                new Token<>(
+                        new byte[4],
+                        new byte[4],
+                        hdfsDelegationTokenKind,
+                        hdfsDelegationTokenService));
+        UserGroupInformation.getCurrentUser().addCredentials(credentials);
+
+        tableEnv.executeSql(
+                String.format(
+                        "create table table2 (x int, y string, z int) partitioned by ("
+                                + " pt_year int, pt_mon string, pt_day string)"
+                                + " tblproperties ('%s' = 'true')",
+                        STREAMING_SOURCE_ENABLE.key()));
```

Comment:
Why do we need to enable streaming source?

Reply:
Removed.